MultiVolumeImporterPlugin.py

```diff
@@ -503,8 +555,15 @@ def load(self,loadable):
     filesPerFrame = nFiles/nFrames
     frames = []
 
-    mvImage = vtk.vtkImageData()
-    mvImageArray = None
+    loadAsVolumeSequence = hasattr(loadable, 'loadAsVolumeSequence') and loadable.loadAsVolumeSequence
+    if loadAsVolumeSequence:
+      volumeSequenceNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLSequenceNode",
+        slicer.mrmlScene.GenerateUniqueName(mvNode.GetName()))
+      volumeSequenceNode.SetIndexName("")
+      volumeSequenceNode.SetIndexUnit("")

```

Comment:
I think it would be nice to initialize those consistently with the multivolume nodes, I think the parallel concept in multivolume are `SetLabelName/SetLabelArray` and the `MultiVolume.FrameIdentifyingDICOMTagUnits` attribute on the MRML node, see: 
* https://github.com/fedorov/MultiVolumeImporter/blob/master/MultiVolumeImporterPlugin.py#L349-L353
* https://github.com/fedorov/MultiVolumeImporter/blob/master/MultiVolumeImporterPlugin.py#L362-L363
* https://github.com/fedorov/MultiVolumeImporter/blob/master/MultiVolumeImporterPlugin.py#L729-L730